pkg/cmd/chat/events/events.go

```diff
@@ -0,0 +1,129 @@
+package events
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/GetStream/stream-cli/pkg/version"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/gorilla/websocket"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{listenCmd()}
+}
+
+func listenCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "listen-events --user-id [user-id]",
+		Short: "Listen to events",
+		Long: heredoc.Doc(`
+			The command opens a WebSocket connection to the backend in the name of the user
+			and prints the received events to the standard output.
+			Press Ctrl+C to exit.
+		`),
+		Example: heredoc.Doc(`
+			# Listen to events for user with id 'my-user-1'
+			$ stream-cli chat listen-events --user-id my-user-1
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			userID, _ := cmd.Flags().GetString("user-id")
+			config := config.GetConfig(cmd)
+			client, _ := config.GetClient(cmd)
+			token, err := client.CreateToken(userID, time.Time{})
+			if err != nil {
+				return err
+			}
+
+			apiKey, _, err := config.GetCredentials(cmd)
+			if err != nil {
+				return err
+			}
+
+			cmd.Println("> 🚨 Warning! The WebSocket connection can be expensive so we close it down after 60 seconds.")
+			time.Sleep(2 * time.Second)
+			// Giving the user 2 seconds to read the warning message
+			// because the first heartbeat is sent super quickly and
+			// takes up the whole screen.
+
+			url := getUrl(userID, apiKey, token)
+			websocket.DefaultDialer.HandshakeTimeout = time.Second * 5
+			conn, _, err := websocket.DefaultDialer.Dial(url, nil)
+			if err != nil {
+				return err
+			}
+			cmd.Println("> Successfully connected. Waiting for events...⌛️")
+
+			exit := make(chan os.Signal, 1)
+			signal.Notify(exit, syscall.SIGINT, syscall.SIGTERM, os.Interrupt)
+
+			// Since keeping connections can be expensive
+			// let's just exit after 60 seconds.
```

Comment:
60 secs seems too low to me

Reply:
don't see how/why this could be expensive per se

Reply:
Yeah I'm not sure, @ferhatelmas explained it in yesterdays standup

Reply:
I think this is mixing of two different things. It was related to MAU increase (money, not perf). 

Here, having a conn open wouldn't be a big deal. We could keep it open until user cancels but still timeout seems reasonable.